sci-libs/libspatialindex/metadata.xml

```diff
@@ -1,5 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<!-- maintainer-needed -->
+	<maintainer type="project">
+		<email>3dprint@gentoo.org</email>
+		<name>Gentoo 3D print</name>
+	</maintainer>
+	<longdescription>
+		Extensible framework for robust spatial indexing
+	</longdescription>
```

Comment:
That's not very long for a `longdescription`. Drop this and rather improve `DESCRIPTION` in the ebuild?

Reply:
OK, longdescription removed, improved description as desired